test/test-enjoi.js

```diff
@@ -292,6 +292,28 @@ Test('types', function (t) {
         });
     });
 
+    t.test('string empty', function (t) {
+        t.plan(2);
+
+        var schemaAllowingEmpty = Enjoi({
+            'type': 'string',
+            'allowEmptyValue': true
+        });
+
+        Joi.validate('', schemaAllowingEmpty, function (error, value) {
+            t.ok(!error,  "Empty string allowed");
+        });
+
+        var defaultStringSchema = Enjoi({
+            'type': 'string'
+        });
+
+        Joi.validate('', defaultStringSchema, function (error, value) {
```

Comment:
Done. Added test for default behaviour with values: `false`, `'invalid'` and `1`.